Automattic/jetpack
# VideoPress: Remove poster images from Calypso media library

Fixes Automattic/greenhouse#960

#### Changes proposed in this Pull Request:

Poster images for Jetpack sites appear in the Calypso media library.
This PR excludes items with the videopress_poster_image meta from the /rest/v1.1/sites/%s/media endpoint

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

* On a sandboxed Jetpack environment with a VideoPress plan
* Before checking out the branch
* Go to your media library and upload a VideoPress video
* Go to wordpress.com/media/<your-jurassic-tube-url>
* You'll see the video and the thumbnails (poster) images
![Capture d’écran 2021-10-26 à 09 36 21](https://user-images.githubusercontent.com/1420594/138831408-d91daf69-9635-463b-954f-e83ced3c8e51.png)
* Checkout this PR
* Reload the Calypso media library
* ✅ VideoPress thumbnails should not be displayed anymore
![Capture d’écran 2021-10-26 à 09 36 38](https://user-images.githubusercontent.com/1420594/138831525-919a5a31-4786-4ad2-8238-38d5737dae47.png)
